js/components/dataset/list.vue

```diff
@@ -21,7 +21,8 @@ export default {
                 key: 'title',
                 sort: 'title',
                 align: 'left',
-                type: 'text'
+                type: 'text',
+                width: 200
```

Comment:
1. What happens if we remove all fields’ `width`s? Doesn't it fit auto-magically?
2. What about using classes/vars for consistency across all admin tables?